src/frontend/sider/Fagsak/FagsakLinje.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import { useLocation } from 'react-router';
+
+import { FileTextIcon, HouseIcon, MagnifyingGlassIcon } from '@navikt/aksel-icons';
+import { Box, Button, HStack } from '@navikt/ds-react';
+
+import type { IMinimalFagsak } from '../../typer/fagsak';
+import type { IPersonInfo } from '../../typer/person';
+import Behandlingsmeny from './Personlinje/Behandlingsmeny/Behandlingsmeny';
+import type { IBehandling } from '../../typer/behandling';
+
+interface FagsaklinjeProps {
+    bruker: IPersonInfo;
+    minimalFagsak: IMinimalFagsak;
+    behandling?: IBehandling;
+}
+
+const aktivFaneStyle = (fanenavn: string) => {
+    const location = useLocation();
+    const urlSplit = location.pathname.split('/');
+    const sluttenPåUrl = urlSplit[urlSplit.length - 1];
+    return sluttenPåUrl === fanenavn ? { textDecoration: 'underline' } : {};
+};
+
+const FagsakLinje: React.FunctionComponent<FagsaklinjeProps> = ({
+    minimalFagsak,
+    bruker,
+    behandling,
+}) => {
```

Comment:
Nitpick: Er vel litt mer vanlig å definere props slik nå om dagen: 

```suggestion
const FagsakLinje = ({
    minimalFagsak,
    bruker,
    behandling,
}: FagsaklinjeProps) => {
```